README.md

```diff
@@ -13,7 +13,7 @@ either in HTML or PDF.
 ## Requirements
 
   - `python3`
-  - `xelatex` (Only for PDF papers)
+  - `texlive-xetex` (Only for PDF papers)
```

Comment:
I think we should keep these, because since this is describing the programs we need rather than the actual package names. For example, non-Ubuntu distros might not call it `texlive-xetex`.
```suggestion
  - `xelatex` (Only for PDF papers)
```